dex/testing/firo/test/electrumx_network_test.go

```diff
@@ -1,8 +1,6 @@
 // This code is available on the terms of the project LICENSE.md file,
 // also available online at https://blueoakcouncil.org/license/1.0.0.
 
-//go:build live
-
```

Comment:
I though live was appropriate. This test needs the internet and takes a long time doesn't it? We can't run it with normal tests automatically.